app.py

```diff
@@ -190,7 +172,9 @@ def _get_commit_sha(user, repository, committish):
     raise typer.Abort()
 
 
-def _side_effects(source: dict, committish: str | None = None):
+def _side_effects(
+    source: dict, committish: str | None = None, original_commit_sha: str = None
```

Comment:
```suggestion
    source: dict, committish: str | None = None, original_commit_sha: str | None = None
```
Extremely minor nitpick you can ignore if you like.

Reply:
TY, fixed and rebased